### ISSUE TITLE
No version in Python package

### ISSUE THREAD

**Issue by michael-burt** [FusionAuth/fusionauth-python-client#3]
```
>>> import fusionauth
>>> fusionauth.__version__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'fusionauth' has no attribute '__version__'
```

**Comment by robotdan** [comment]
@tyduptyler13 can you take a look when you get a moment? Thanks.

**Comment by tyduptyler13** [comment]
Fixed in `1.7.2` release on PyPi
